src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -360,7 +431,7 @@ EncodedResults StaticLLMPipeline::generate(
 
     auto padded_input_ids = m_prefill_request.get_tensor("input_ids");
     const size_t offset = padded_input_ids.get_size() - input_ids.get_size();
-    copy_with_offset(input_ids, offset, padded_input_ids);
+    copy_with_offset(input_ids, int32_t(offset), padded_input_ids);
```

Comment:
hmm, strange cast. Can `offset` be in the right type? Or at least `static_cast`

Reply:
Used `static_cast`. But, yes, `offset` can be of proper type, but it will just require `static_cast` in another calls to cast it back to `size_t`